telco-hub/configuration/reference-crs/required/acm/observabilitySecret.yaml

```diff
@@ -11,5 +11,4 @@ metadata:
   namespace: open-cluster-management-observability
 type: kubernetes.io/dockerconfigjson
 data:
-  .dockerconfigjson:
-  # <REDACTED>
+  .dockerconfigjson: '{{- if eq (lookup "v1" "Secret" "open-cluster-management" "multiclusterhub-operator-pull-secret").kind "Secret" -}} {{- fromSecret "open-cluster-management" "multiclusterhub-operator-pull-secret" ".dockerconfigjson" -}} {{- else -}} {{- fromSecret "openshift-config" "pull-secret" ".dockerconfigjson" -}} {{- end -}}'
```

Comment:
In the Hub RDS we don't build policy out of the reference CRs (that is why we include a couple fully built policies as you have below). That means this template will never be evaluated/used. For clarity I'd suggest this:
```suggestion
  .dockerconfigjson: ''  # Value provided by user or by pull-secret-openshift-config-copy policy
```